create-users-projects.yml

```diff
@@ -2,9 +2,9 @@
 - name: Create users and projects
   hosts: localhost
   roles:
-    - role: users-projects
-      tags: ['create-hosts']
-      os_cloud: opentech-sjc
+    - role: cloud-bootstrap
+      tags: ['accounts']
```

Comment:
This tag is inappropriate, as it applies the `accounts` tag to the entire role, and inside that role, both accounts _and_ networks are created. It'd be best to remove the tag here all together, and just rely on the tags inside the `main.yml` file within the role.